lib/hanami_context_logging/formatter/with_context.rb

```diff
@@ -17,8 +17,13 @@ def formatted_contexts
     end
 
     def contexts
+      provider_context = if @context_provider.context.is_a?(Hash)
+                           @context_provider.context
+                         else
+                           @context_provider.context.to_h
```

Comment:
Maybe we can always call `to_h` since `Hash#to_h` returns itself

Reply:
that's what @tsungtingdu and I also mentioned, but I thought this is more intentional :keke: